product_docs/docs/tpa/23/architecture-M1.mdx

```diff
@@ -57,28 +57,27 @@ The tables below describe the mandatory options for M1
 and additional important options.
 More detail on the options is provided in the following section.
 
-#### Mandatory Options
+#### Mandatory options
 
 | Parameter                                             | Description                                                                               |
 | ----------------------------------------------------- | ----------------------------------------------------------------------------------------- |
 | `--architecture` (`-a`)                               | Must be set to `M1`.                                                                      |
-| Postgres flavour and version (e.g. `--postgresql 15`) | A valid [flavour and version specifier](tpaexec-configure/#postgres-flavour-and-version). |
+| Postgres flavor and version (e.g., `--postgresql 15`) | A valid [flavor and version specifier](tpaexec-configure/#postgres-flavor-and-version). |
 
-<br/><br/>
 
-#### Additional Options
+#### Additional options
 
-| Parameter                 | Description                                                                | Behaviour if omitted                                                                                 |
+| Parameter                 | Description                                                                | Behavior if omitted                                                                                 |
 | ------------------------- | -------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------- |
 | `--platform`              | One of `aws`, `docker`, `bare`.                                            | Defaults to `aws`.                                                                                   |
 | `--num-cascaded-replicas` | The number of cascaded replicas from the first replica.                    | Defaults to 1.                                                                                       |
-| `--enable-efm`            | Configure Failover Manager as the cluster failover manager. | TPA will select EFM<br/>as the failover manager<br/>for EPAS, and repmgr<br/>for all other flavours. |
+| `--enable-efm`            | Configure Failover Manager as the cluster failover manager. | TPA selects EFM as the failover manager for EPAS and repmgr for all other flavors. |
```

Comment:
The breaks were deliberatly added to compensate for Markdown's table balancing on the page. 

```suggestion
| `--enable-efm`            | Configure Failover Manager as the cluster failover manager. | TPA selects EFM<br/>as the failover manager<br/>for EPAS and repmgr<br/>for all other flavors. |
```